src/screens/TopicPageScreen/index.js

```diff
@@ -1,34 +1,33 @@
-import * as React from 'react';
+import {useNavigation, useRoute} from '@react-navigation/native';
 import _ from 'lodash';
+import * as React from 'react';
+import {Animated, Platform, StyleSheet, View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
 import SimpleToast from 'react-native-simple-toast';
-import {Animated, Platform, StyleSheet} from 'react-native';
-import {SafeAreaProvider, useSafeAreaInsets} from 'react-native-safe-area-context';
-import {useNavigation, useRoute} from '@react-navigation/native';
 
 import BlockComponent from '../../components/BlockComponent';
 import ButtonAddPostTopic from '../../components/Button/ButtonAddPostTopic';
-import MemoizedListComponent from './MemoizedListComponent';
-import NavHeader from './elements/NavHeader';
-import ShareUtils from '../../utils/share';
 import TiktokScroll from '../../components/TiktokScroll';
-import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
-import dimen from '../../utils/dimen';
-import removePrefixTopic from '../../utils/topics/removePrefixTopic';
-import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
-import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
+import {Context} from '../../context';
+import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
 import useOnBottomNavigationTabPressHook, {
   LIST_VIEW_TYPE
 } from '../../hooks/navigation/useOnBottomNavigationTabPressHook';
-import {Context} from '../../context';
-import {downVote, upVote} from '../../service/vote';
 import {getFeedDetail} from '../../service/post';
 import {getTopicPages} from '../../service/topicPages';
 import {getTopics, getUserTopic} from '../../service/topics';
-import {getUserId} from '../../utils/users';
-import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
+import {downVote, upVote} from '../../service/vote';
+import dimen from '../../utils/dimen';
 import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
-import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
-import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
+import ShareUtils from '../../utils/share';
+import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
+import removePrefixTopic from '../../utils/topics/removePrefixTopic';
+import {getUserId} from '../../utils/users';
+import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
+import MemoizedListComponent from './MemoizedListComponent';
+import NavHeader from './elements/NavHeader';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reorganized and `SafeAreaProvider` has been removed. This change is consistent with the PR summary, which mentions replacing `SafeAreaProvider` with `SafeAreaView`. However, it's important to note that `SafeAreaView` is not imported in this file. If it's used later in the code, it could cause a reference error. Please ensure that all necessary imports are included.
```diff
- import {SafeAreaProvider, useSafeAreaInsets} from 'react-native-safe-area-context';
+ import {useSafeAreaInsets} from 'react-native-safe-area-context';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->